### ISSUE TITLE
[MET Éireann] Use XML API (more data)

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#787]
Eventually, we should replace the currently used API with the regular XML because it has more data:

https://data.gov.ie/dataset/met-eireann-forecast-api

However, it's harder to parse (and we never parsed XML so far).

**Comment by Feakster** [comment]
I was going to open a new issue log, but I noticed that this ones already open.

I think one of the main things missing from the current API (that is present in the API that returns XML, & in the APIs of the other weather sources) is hour-by-hour precipitation probability.

**Comment by papjul** [comment]
Yes, that's exactly what feedback I got, but I also noticed other details are in the XML. It's, however, not very convenient to process as precipitation data is on a separate object compared to other details.

I wish they could update to v2.0 of the MET No standard so we would just have a json file easy to process.

**Comment by Feakster** [comment]
Is a version 2 in the works?

**Comment by papjul** [comment]
They use version 1.9 of the XML schema of MET Norway which was deprecated on 2020-12-01.

The newer schema can be found here: https://api.met.no/weatherapi/locationforecast/2.0/schema

Having a JSON format would also be a nice addition.

**Comment by papjul** [comment]
WIP PR on #1357 

Unfortunately:
> Documentation says:
> 
> ```
> URLs of the form:
> http://metwdb-openaccess.ichec.ie/metno-wdb2ts/locationforecast
> should be replaced with
> http://openaccess.pf.api.met.ie/metno-wdb2ts/locationforecast
> ```
> 
> However, the TLS certificate used on `openaccess.pf.api.met.ie` is the one from `metwdb-openaccess.ichec.ie` which is invalid, so cannot progress further for the time being

**Comment by papjul** [comment]
Canceling as they didn’t care about fixing their TLS certificate in more than a year.

Please reopen only if the API can be accessed with a valid TLS certificate on endpoints using https.
